src/index.js

```diff
@@ -59,9 +65,19 @@ function IpfsAPI (host_or_multiaddr, port) {
 
   // -- Interface
 
+  /** @public */
   self.send = requestAPI
 
-  self.add = (files, opts, cb) => {
+  /**
+   * Add a file/many files to IPFS returning the hash and name. The
+   * name value will only be set if you are actually sending a file.
+   *
+   * @public
+   * @param {} files
+   * @param {object} opts
+   * @param {function} cb
+   */
```

Comment:
my opinion: in general, while in other languages I do like godoc, i find that most successful modules I use in javascript are very small, easy to use, and have the docs/examples directly in the readme. i typically run away from jsdoc-ed modules.